dev/email-field.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <script type="module">
+      import '@vaadin/email-field';
+    </script>
+  </head>
+
+  <body>
+    <vaadin-email-field label="Required" required></vaadin-email-field>
+
+    <div>Is valid? <span id="validity"></span></div>
+
+    <script>
+      const field = document.querySelector('vaadin-email-field');
+      const validity = document.querySelector('#validity');
+
+      field.addEventListener('change', (e) => {
```

Comment:
The validity of the `vaadin-email-field` component changes when the field loses focus and so this listener runs before the validity of the field is changed, causing it to be out of sync. Changing it to listen to the `focusout` event fixes it. If we want validation on `change`, then the component must be updated to update its validity on `change`, or to fire the `change` before the `focusout` event is fired.
```suggestion
      field.addEventListener('focusout', (e) => {
```

Reply:
This was reported in https://github.com/vaadin/web-components/issues/2517 and fixed in https://github.com/vaadin/web-components/pull/2519

Reply:
Well, that's great. Then, other than my suggestion for the titles, the PR looks good to me.

Reply:
Removed this listener.